tests/Functional/TranslationTestBase.php

```diff
@@ -36,6 +36,7 @@ class TranslationTestBase extends BrowserTestBase {
     'options',
     'oe_translation',
     'oe_translation_test',
+    'oe_translation_metatag_test',
```

Comment:
Do we really need to enable oe_translation_metatag_test for all Functional tests?
Maybe we will do enabling in tests where we really do testing?

Reply:
It was, why would we change now?

Reply:
It is an optional comment. As far as we did reorganising of dependencies I just throught that it would be reasonable also to simplify dependencies in tests.